draft-ietf-taps-interface.md

```diff
@@ -1904,47 +1904,70 @@ options.
-  less-than-best-effort treatment, or to assign the traffic to a lower-effort service.
-
+  : The application makes no representation about its expected capacity
+  profile. No special optimizations of the tradeoff between delay, delay
+  variation, and bandwidth efficiency should be made when selecting and
+  configuring transport protocol stacks. When the underlying transport protocol
+  stack(s) support per-connection DSCP signaling, transmitted traffic SHOULD carry
+  the DSCP Best Effort PHB. \[EDITOR'S NOTE: cite]
+
+  Scavenger/Bulk: 
+  : The application is not interactive. It expects to send
+  and/or receive a large amount of data, without any urgency. This can, for
+  example, be used to select protocol stacks with scavenger transmission control
+  and/or to assign the traffic to a lower-effort service. When the underlying
+  transport protocol stack(s) support per-connection DSCP signaling, transmitted
+  traffic SHOULD carry the DSCP Less than Best Effort PHB. \[EDITOR'S NOTE: cite]
+
+  Low Latency/Interactive: : The application is interactive, and prefers loss to
+  latency. Response time should be optimized at the expense of bandwidth
+  efficiency and delay variation when sending on this connection. This can be
+  used by the system to disable the coalescing of multiple small Messages into
+  larger packets (Nagle's algorithm); to prefer immediate acknowledgment from
+  the peer endpoint when supported by the underlying transport; and so on. When
+  the underlying transport protocol stack(s) support per-connection DSCP
+  signaling, transmitted traffic SHOULD carry the DSCP Expedited Forwarding PHB.
+  \[EDITOR'S NOTE: cite]
+
+  Low Latency/Non-Interactive: 
+  : The application prefers loss to latency but is
+  not interactive. Response time should be optimized at the expense of bandwidth
+  efficiency and delay variation when sending on this connection. When the
+  underlying transport protocol stack(s) support per-connection DSCP signaling,
+  transmitted traffic SHOULD carry a DSCP Assured Forwarding
+  (AF21,AF22,AF23,AF24) PHB. \[EDITOR'S NOTE: cite]
+
+  Constant-Rate Streaming: 
+  : The application expects to send/receive data at a
+  constant rate after Connection establishment. Delay and delay variation should
+  be minimized at the expense of bandwidth efficiency. This implies that the
+  Connection may fail if the desired rate cannot be maintained across the Path.
+  A transport may interpret this capacity profile as preferring a circuit
+  breaker {{?RFC8084}} to a rate-adaptive congestion controller. When the
+  underlying transport protocol stack(s) support per-connection DSCP signaling,
+  transmitted traffic SHOULD carry a DSCP Assured Forwarding
+  (AF31,AF32,AF33,AF34) PHB.
+
+  High Throughput Data: : \[EDITOR'S NOTE: Gorry wants this, but bht has no idea
+  how to implement it beyond DSCP AF2n. Cut, or specity how this is really
+  different from one of the other profiles.]
```

Comment:
What a strange beast. Oh well. It exists and there's probably a point. As for implementation, see:  https://trac.tools.ietf.org/html/rfc4594#section-4.8   AF1n, it seems, not AF2n.

Reply:
will change to AF1n, then wait for @gorryfair to tell me what it means.